components/uid-text/src/uid-element.ts

```diff
@@ -0,0 +1,87 @@
+import { LitElement, css, CSSResultGroup } from 'lit';
```

Comment:
:x: We really duplicate this class each time ? I think it's better to externalize it now (put a ref on npm repo, put this class in common folder and reference it when we need it)

Reply:
In a next step we will extract this classe as a stand alone Npm module